android/src/main/java/com/tanguyantoine/react/MusicControlModule.java

```diff
@@ -438,6 +453,15 @@ synchronized public void enableControl(String control, boolean enable, ReadableM
         session.setPlaybackState(state);
     }
 
+    private void addBitmapToMemoryCache(String key, Bitmap bitmap) {
+        if (getBitmapFromMemCache(key) == null) {
+            mMemoryCache.put(key, bitmap);
+        }
+    }
+
+    private Bitmap getBitmapFromMemCache(String key) {
+        return mMemoryCache.get(key);
+    }
```

Comment:
These methods are pretty useless, they only add up to Android's method count and can be easily replaced by direct calls instead